src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_utils.py

```diff
@@ -29,7 +29,7 @@ def test_resource_group_level_id(self):
         )
 
         self.assertEqual(scope, f"/subscriptions/{subscription_id}/resourceGroups/test-what-if-rg")
-        self.assertEqual(relative_resource_id, f"Microsoft.Sql/servers/dbserver")
+        self.assertEqual(relative_resource_id, f"Microsoft.Sql/servers/dbserver")   # noqa: F541
```

Comment:
Same here, remove `f`:
```suggestion
        self.assertEqual(relative_resource_id, "Microsoft.Sql/servers/dbserver")
```